client/reader/following/controller.js

```diff
@@ -7,31 +9,35 @@ const analyticsPageTitle = 'Reader';
 
 const exported = {
 	followingManage( context, next ) {
-		const basePath = sectionify( context.path );
-		const fullAnalyticsPageTitle = analyticsPageTitle + ' > Manage Followed Sites';
-		const mcKey = 'following_manage';
-		const { q: sitesQuery, s: subsQuery, sort: subsSortOrder, showMoreResults } = context.query;
+		if ( isEnabled( 'subscription-management-redirect-following' ) ) {
+			page.redirect( '/read/subscriptions' );
+		} else {
+			const basePath = sectionify( context.path );
+			const fullAnalyticsPageTitle = analyticsPageTitle + ' > Manage Followed Sites';
+			const mcKey = 'following_manage';
+			const { q: sitesQuery, s: subsQuery, sort: subsSortOrder, showMoreResults } = context.query;
 
-		setPageTitle( context, i18n.translate( 'Manage followed sites' ) );
+			setPageTitle( context, i18n.translate( 'Manage followed sites' ) );
 
-		trackPageLoad( basePath, fullAnalyticsPageTitle, mcKey );
+			trackPageLoad( basePath, fullAnalyticsPageTitle, mcKey );
 
-		context.primary = (
-			<AsyncLoad
-				require="calypso/reader/following-manage"
-				key="following-manage"
-				initialFollowUrl={ context.query.follow }
-				sitesQuery={ sitesQuery }
-				subsQuery={ subsQuery }
-				showMoreResults={ Boolean( showMoreResults ) }
-				subsSortOrder={ subsSortOrder }
-				context={ context }
-			/>
-		);
+			context.primary = (
+				<AsyncLoad
+					require="calypso/reader/following-manage"
+					key="following-manage"
+					initialFollowUrl={ context.query.follow }
+					sitesQuery={ sitesQuery }
+					subsQuery={ subsQuery }
+					showMoreResults={ Boolean( showMoreResults ) }
+					subsSortOrder={ subsSortOrder }
+					context={ context }
+				/>
+			);
+		}
 		next();
```

Comment:
It looks to me like the `next();` can also be included in the else, and/or you can simply return on line 13, this would save the need for the explicit `else` statement too.